### ISSUE TITLE
AttributeError: 'module' object has no attribute 'sysconfig'  

### ISSUE THREAD

**Issue by baeuml** [gregjurman/tesserwrap#10]
This is with Python 2.7.3 on a standard Ubuntu 12.04:

``` python
In [1]: from tesserwrap import Tesseract
---------------------------------------------------------------------------                                                                              
AttributeError                            Traceback (most recent call last)                                                                              
/home/baeuml/<ipython-input-1-b91b678fa26c> in <module>()                                                                                                
----> 1 from tesserwrap import Tesseract                                                                                                                 

/usr/local/lib/python2.7/dist-packages/tesserwrap-0.1.1-py2.7-linux-x86_64.egg/tesserwrap/__init__.py in <module>()                                      
----> 1 from .core import tr                                                                                                                             
      2 from ctypes import c_ulonglong, byref                                                                                                            
      3 import sys                                                                                                                                       
      4 import warnings                                                                                                                                  
      5                                                                                                                                                  

/usr/local/lib/python2.7/dist-packages/tesserwrap-0.1.1-py2.7-linux-x86_64.egg/tesserwrap/core.py in <module>()                                          
     68                                                                                                                                                  
     69                                                                                                                                                  
---> 70 tr = load_library('libtesserwrap', os.path.dirname(__file__))                                                                                    
     71                                                                                                                                                  
     72                                                                                                                                                  

/usr/local/lib/python2.7/dist-packages/tesserwrap-0.1.1-py2.7-linux-x86_64.egg/tesserwrap/core.py in load_library(libname, loader_path)                  
     45                                                                                                                                                  
     46     """                                                                                                                                          
---> 47     so_ext = get_shared_lib_extension()                                                                                                          
     48     libname_ext = [libname + so_ext]                                                                                                             
     49     if sys.version[:3] >= '3.2':                                                                                                                 

/usr/local/lib/python2.7/dist-packages/tesserwrap-0.1.1-py2.7-linux-x86_64.egg/tesserwrap/core.py in get_shared_lib_extension(is_python_ext)             
     28                                                                                                                                                  
     29     """                                                                                                                                          
---> 30     so_ext = distutils.sysconfig.get_config_var('SO') or ''                                                                                      
     31     # fix long extension for Python >=3.2, see PEP 3149.                                                                                         

     32     if not is_python_ext and 'SOABI' in distutils.sysconfig.get_config_vars():                                                                   

AttributeError: 'module' object has no attribute 'sysconfig'
```

**Comment by jmel** [comment]
I am also getting this problem. I am running on macosx 10.8

**Comment by gregjurman** [comment]
Sorry for the extreme delay on getting to this. Line should be fixed in the repo. Patch is not yet out on PyPI until I fix jmel's issue with OSX (#11).

**Comment by gregjurman** [comment]
This is fixed in master branch, new build will be put on PyPI on Friday/Saturday.
